httpx/_client.py

```diff
@@ -107,6 +110,17 @@ def _get_proxy_map(
             proxy = Proxy(url=proxies) if isinstance(proxies, (str, URL)) else proxies
             return {"all": proxy}
 
+    @property
+    def base_url(self) -> URL:
+        """
+        Base URL to use when sending requests with relative URLs.
+        """
+        return self._base_url
```

Comment:
Where is `self._base_url` set/created? I see `self.base_url` on [Line 72](https://github.com/encode/httpx/pull/1130/files#diff-eff812167840d2389ea5f0d809b30c32R72), perhaps that should be privatized?

Reply:
Thanks yup. Should have been `self._base_url` in `__init__`.
It was still working (because there's a property setter) but it's neater now.